### PR TITLE
android-intro anim

### PR DESCRIPTION
**Pull request by kjanderson2**
Animation for intro page

### CODE REVIEW

File: Android/app/src/main/res/layout/fragment_intro.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    android:background="@color/black"
+    app:layoutDescription="@xml/intro_motion_scene">
+
+    <androidx.constraintlayout.utils.widget.ImageFilterView
+        android:id="@+id/hand_watermark"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:src="@drawable/hand_watermark"
+        app:altSrc="@drawable/hand_watermark_red"
+        android:background="@drawable/hand_watermark" />
```
Comment (Mehequanna) [inline Android/app/src/main/res/layout/fragment_intro.xml:16]: We don't need this background. This is what's causing the double hand.

File: Android/app/src/main/res/layout/fragment_intro.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    android:id="@+id/intro_motion_layout"
+    android:background="@color/black"
+    app:layoutDescription="@xml/intro_motion_scene">
+
+    <androidx.constraintlayout.utils.widget.ImageFilterView
+        android:id="@+id/hand_watermark"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:src="@drawable/hand_watermark"
+        app:altSrc="@drawable/hand_watermark_red"
```
Comment (Mehequanna) [inline Android/app/src/main/res/layout/fragment_intro.xml:15]: We can remove this line and the resource if you are ok with my other suggestion in the Motion Layout.

File: Android/app/src/main/res/xml/intro_motion_scene.xml [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            motion:visibilityMode="ignore">
+            <Layout
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                motion:layout_constraintStart_toStartOf="parent"
+                motion:layout_constraintEnd_toEndOf="parent"
+                motion:layout_constraintBottom_toBottomOf="parent" />
+            <CustomAttribute
+                motion:attributeName="crossfade"
+                motion:customFloatValue="0.0" />
```
Comment (Mehequanna) [inline Android/app/src/main/res/xml/intro_motion_scene.xml:23]: If we remove the background and the `altSrc`, we could change this to:
```
            <CustomAttribute
                motion:attributeName="ColorFilter"
                motion:customColorValue="#272727" />
```

File: Android/app/src/main/res/xml/intro_motion_scene.xml [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+            motion:visibilityMode="ignore">
+            <Layout
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                motion:layout_constraintStart_toStartOf="parent"
+                motion:layout_constraintEnd_toEndOf="parent"
+                motion:layout_constraintBottom_toBottomOf="parent" />
+            <CustomAttribute
+                motion:attributeName="crossfade"
+                motion:customFloatValue="1.0" />
```
Comment (Mehequanna) [inline Android/app/src/main/res/xml/intro_motion_scene.xml:143]:
```
<CustomAttribute
                motion:attributeName="ColorFilter"
                motion:customColorValue="#81170E" />
```

**Review by Mehequanna — APPROVED** [review]
This is incredible work. I was trying to learn more about the MotionLayout for the presentation coming up. In the process, I noticed a couple things. I'm curious what you think about the color change idea.

Again this was amazing. I'm blown away with how good you are with motionLayout.

File: Android/app/src/main/res/xml/intro_motion_scene.xml (hunk 3 above)
Comment (kjanderson2) [reply Android/app/src/main/res/xml/intro_motion_scene.xml]: Awesome! This is much better!

**Pull request merged**